net/hysteria/files/hysteria.config

```diff
@@ -0,0 +1,18 @@
+config server 'server'
+	option enabled '0'
+
+config client 'client'
+	option enabled '0'
+
+	option enabled_tproxy '1'
+	option tproxy_mark "0x1000"
+	option tproxy_user "hysteria"
+	option tproxy_port "2500"
+	option tproxy_table "client"
+	option tproxy_set "proxy"
+	option tproxy_route_table "100"
+
+	option enabled_dnsmasq '1'
+	option dnsmasq_server "127.0.0.53"
```

Comment:
127.0.0.1#53 ?

Reply:
Seems more correct although technically the whole 127/8 range is available.